ccd-definition/ComplexTypes.json

```diff
@@ -1271,18 +1271,100 @@
     "ListElementCode": "orderType",
     "FieldType": "MultiSelectList",
     "FieldTypeParameter": "OrderType",
-    "ElementLabel": "Which orders and directions do you want to apply for?",
+    "ElementLabel": "Which orders do you need?",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "Orders",
-    "ListElementCode": "directionsAndInterim",
+    "ListElementCode": "emergencyProtectionOrderLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Emergency protection order",
+    "FieldShowCondition": "orderType CONTAINS \"EMERGENCY_PROTECTION_ORDER\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Orders",
+    "ListElementCode": "emergencyProtectionOrders",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "EmergencyProtectionOrderType",
+    "ElementLabel": "Do you need any of these related orders?",
+    "FieldShowCondition": "orderType CONTAINS \"EMERGENCY_PROTECTION_ORDER\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Orders",
+    "ListElementCode": "emergencyProtectionOrderDetails",
     "FieldType": "TextArea",
-    "ElementLabel": "Directions and interim orders, if needed",
+    "ElementLabel": "Give details",
+    "FieldShowCondition": "emergencyProtectionOrders CONTAINS \"EMERGENCY_PROTECTION_ORDER_OTHER\" AND orderType CONTAINS \"EMERGENCY_PROTECTION_ORDER\"",
     "SecurityClassification": "Public"
-  }
-,
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Orders",
+    "ListElementCode": "emergencyProtectionOrderDirection",
```

Comment:
Please rename to be `emergencyProtectionOrderDirections` (plural) as it holds an array of directions.